spectral-ruleset-govuk-public/ruleset.yaml

```
@@ -1,2 +1,12 @@
 extends:
   - [spectral:oas, all]
+rules:
+  semver:
+    severity: error
+    recommended: true
+    message: Version should use semantic versioning. {{value}} is not a valid version.
+    given: $.info.version
+    then:
+      function: pattern
+      functionOptions:
+        match: "^([0-9]+.[0-9]+.[0-9]+)$"
```

I could continue to have the tests passing with the following regex:

```suggestion
        match: "^([0-9].[0-9].[0-9])$"
```

Should we add a more complex number i.e. `10.0.500`?